cvat-ui/src/containers/annotation-page/standard-workspace/objects-side-bar/object-buttons.tsx

```diff
@@ -44,6 +46,7 @@ function mapStateToProps(state: CombinedState, own: OwnProps): StateToProps {
             player: {
                 frame: { number: frameNumber },
             },
+            editing: editedState,
```

Comment:
It is better to pass exact boolean property of the object you only use in this component. Potentially it will cause less renderings